rudaux/flows.py

```diff
@@ -178,7 +193,7 @@ def build_autoext_flows(config, args):
                 submission_sets = subm.build_submission_set.map(unmapped(config), submission_sets)
 
                 # Compute override updates
-                overrides = subm.get_latereg_overrides.map(unmapped(config.latereg_extension_days[group]), submission_sets)
+                overrides = subm.get_latereg_overrides.map(unmapped(config.latereg_extension_days[group]), submission_sets, config)
```

Comment:
you need to `unmapped` the config (prefect uses `.map` to dynamically create task graphs at runtime, but to use this ability you have to explicitly "unmap" the non-mapped inputs)